packages/app-store/EventTypeAppContext.tsx

```diff
@@ -5,13 +5,20 @@ export type GetAppData = (key: string) => unknown;
 export type SetAppData = (key: string, value: unknown) => void;
 type LockedIcon = JSX.Element | false | undefined;
 type Disabled = boolean | undefined;
+
+type AppContext = {
+  getAppData: GetAppData;
+  setAppData: SetAppData;
+  LockedIcon: LockedIcon;
+  disabled: Disabled;
+};
 // eslint-disable-next-line @typescript-eslint/no-empty-function
-const EventTypeAppContext = React.createContext<[GetAppData, SetAppData, LockedIcon, Disabled]>([
-  () => ({}),
-  () => ({}),
-  undefined,
-  undefined,
-]);
+const EventTypeAppContext = React.createContext<AppContext>({
+  getAppData: () => ({}),
+  setAppData: () => ({}),
+  LockedIcon: undefined,
+  disabled: undefined,
```

Comment:
I think we can remove this undefined assignment now. Earlier it was needed because it was an array and it has to have all the items for clarity.

Reply:
Fixed it.